src/parse.d

```diff
@@ -7471,11 +7459,10 @@ public:
 
     Expression parseCmpExp()
     {
-        Expression e;
         Expression e2;
```

Comment:
D allows to declare variables directly under case statements, without conflict with same name variables in other cases. So you can also declare `e2` in each cases.